src/Observables.jl

```diff
@@ -365,7 +368,7 @@ end
 
 struct MapUpdater{F, T} <: Function
     f::F
-    observable::Observable{T}
+    observable::AbstractObservable{T}
```

Comment:
I think this should  ideally stay concretely typed... Can we move this to the type parameter without problems?